### PR TITLE
Use Mesos API for logs instead of Logdriver

### PR DESCRIPTION
**Pull request by smn**

### CODE REVIEW

**Comment by smn** [comment]
ready for review.

File: mc2/controllers/base/templates/app_logs.html [hunk 1]
```diff
@@ -2,42 +2,46 @@
...
 <div id="content-main" class="new-project">
     <div class="box-header with-border">
       <h3 class="box-title">Logs for <a href="{% url 'home' %}#{{controller.pk}}">{{controller.name|capfirst}}</a></h3>
     </div>
-    <div id="logs">
+    {% for path in paths %}
+    <div class="box-header with-border">
+        <h4 class="box-title">{{path}}</h4>
+        <p>
+            Download the <a href="{% url "mesos_file_log_view" task_id=task_id path=path %}?download=true">full log file</a>.
```
Comment (miltontony) [inline mc2/controllers/base/templates/app_logs.html]: This will be handy!

**Comment by miltontony** [comment]
👍 🍰 Ship it!

**Comment by smn** [comment]
#103 fixes some more issues coming out of this PR.

**Pull request merged**